src/main/kotlin/com/trivigenai/models/TriviaRound.kt

```diff
@@ -0,0 +1,22 @@
+package com.trivigenai.models
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class TriviaQuestion(
+    val question: String,
+    val options: List<String>,
+    val answer: Int
+)
+
+@Serializable
+sealed class Round {
+    data class TriviaRound(
```

Comment:
Looks like `@Serializable` is needed for the subtypes as well. I just tested it.